packages/react/package.json

```diff
@@ -10,17 +16,14 @@
     "lib"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: run tests in xing/hops repo\" && exit 1"
   },
-  "keywords": [],
-  "author": "",
-  "license": "MIT",
   "peerDependencies": {
     "hops-config": "6.1.0",
-    "react": "^15.5.4",
+    "react": "^15.6.1",
```

Comment:
That would make sense - but, seeing that we only ever test on the latest versions of our dependencies, it would give us quite a blind spot. Obviously, we could simply rely on upstream's interpretation and implementation of semver... 🤷‍♂️

Let's discuss this irl.